File: year2020/month10/day1029/ms_1029.cs
using System;

namespace algorithm_89
{
    class ms_1029
    {
        static void Main(string[] args)
        {
            String input = Console.ReadLine();
            int strLen = input.Length;
            
            Console.WriteLine();

            if(strLen <= 100) 
            {
                for(int i = strLen-1; i >= 0; i--) 
                {
                    for(int j = i; j < strLen; j++)
                        Console.Write(input[j]);

                    for(int j = 0; j < i; j++)
                        Console.Write(input[j]);

                    Console.WriteLine();
                }
            }
        }
    }
}

File: year2020/month10/day1029/ms_1029.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
	string input;

	getline(cin, input);

	int strlen = input.length();

	cout << endl;

	if (strlen <= 100) {
		for (int i = strlen - 1; i >= 0; i--) {
			for (int j = i; j < strlen; j++)
				cout << input[j];

			for (int j = 0; j < i; j++)
				cout << input[j];

			cout << endl;
		}
	}

	return 0;
}